src/System.Text.Json/src/System/Text/Json/Reader/JsonReaderHelper.Date.cs

```diff
@@ -350,6 +374,8 @@ public static bool TryGetNextTwoDigits(ReadOnlySpan<byte> source, out int value)
             return true;
         }
 
+        // The following methods are borrowed verbatim from src/Common/src/CoreLib/System/Buffers/Text/Utf8Parser/Utf8Parser.Date.Helpers.cs
```

Comment:
How do we plan to keep these sources in sync? Would it be easier if we just included a reference to that source file in the csproj? For example, like we do in System.Memory csproj?
https://github.com/dotnet/corefx/blob/811e41aa65b5025846defe2b8b886bba23328198/src/System.Memory/src/System.Memory.csproj#L63

Reply:
These helpers are private to the `Utf8Parser` class. Filed an issue to extract them to a static helper class https://github.com/dotnet/corefx/issues/35953.